src/plugins/tables/tables.js

```diff
@@ -84,6 +84,29 @@ var componentName = "wb-tables",
 						dataTableExt = $.fn.dataTableExt;
 
 					/*
+					* Add printing support
+					*/
+					if ( $elm.hasAttribute( "print" ) ) {
+						var $btn = $( "<button class='btn btn-default mrgn-bttm-md' id='" + elmId + "_datatable-print'>Print Table</button>" );
```

Comment:
For the print button text, see how the variable "i18nText"  is used in the init function of the data table javascript. The internationalized text are located in a big csv file here https://github.com/wet-boew/wet-boew/tree/master/src/i18n